lua/harpoon/utils.lua

```diff
@@ -1,10 +1,19 @@
 local Path = require("plenary.path")
 local data_path = vim.fn.stdpath("data")
+local project_key = vim.loop.cwd()
```

Comment:
Only feedback is lets make `project_key` a function and return current `vim.loop.cwd`.  If we do not, then we will get out of date harpoon menus when working with git worktrees.

btw, this change you want could also be done via git worktrees :)

Reply:
Thanks for the feedback

I would like to use git worktrees, however, at work we mount our project folder to a docker container. Working with different folders for different branches would not allow me to jump branches as easily while keeping the same docker container running. I have no experience with git worktrees, so correct me if I'm wrong 😄